gmail-to-salesforce-lead/README.md

```diff
@@ -0,0 +1,80 @@
+
+When the user receives an email pertaining to a lead, they will mark that thread with a specific label (e.g. `"Lead"`). This Ballerina program will run continuously in the background, polling the email server every 10 minutes for threads marked with this label. If an email is found, its content will be read and used to infer the following details
+* First name
+* Last name
+* Phone number
+* Email address
+* Company name
+* Designation
+
+Once these details have been inferred, a new lead will be generated on Salesforce.
+
+![Flow diagram](/docs/flow-diagram.png)
+
+## Prerequisites
+* An email account configured to use Gmail
+* An account on the Google Cloud Platform
+* An OpenAI account with API usage enabled
+* A Salesforce account
+
+>Note: The following steps will require you to generate keys for the Gmail, OpenAI and Salesforce APIs. These keys will have to be securely stored in the `Config.toml` file in the project directory under the relevant fields. To see a template of this file, please see the [`Config.toml.example`] file (./Config.toml.example).
+
+### Configuring your email account to use Gmail
+> Note: If you already have a Gmail account (ending with `@gmail.com`) or your acccount is on the Google workspace, you do not need to follow the steps below. In essence, if you can access your email via `www.gmail.com`, the following is not necessary.
+1. Visit [Gmail](https://gmail.com) and create a new account or log into an existing account.
+2. Enter the `Accounts` tab under settings and click on `Add a mail account`.
+3. Provide the necessary authentication details to your email account.
+4. After adding a mail account, you should be able to see all new emails received to your email via the Gmail interface.
+
+### Obtaining the Gmail API keys
+1. Create a new [Google Cloud Platform project](https://console.cloud.google.com). 
+2. Find and click `APIs & Services` --> `Library` from the navigation menu.
+3. In the searchbox enter `"Gmail"`.
+4. Then select Gmail API and click `Enable` button.
+5. Complete OAuth consent screen setup.
+6. Click the `Credential` tab from the left sidebar. In the displaying window click on the `Create Credentials` button Select OAuth client ID.
+7. Fill in the required fields. Add `"https://developers.google.com/oauthplayground"` to the Redirect URI field.
+8. Note down the `clientId` and `clientSecret`.
+9. Visit https://developers.google.com/oauthplayground/. Go to settings (Top right corner) -> Tick 'Use your own OAuth credentials' and insert Oauth ClientId and clientSecret. Click close.
+10. Then, Complete Step1 (Select and Authorize APIs)
+11. Make sure you select the `"https://www.googleapis/auth/gmail.modify"` and `"https://www.googleapis/auth/gmail.labels` OAuth scopes. These two scopes will allow the program to read emails including adding/removing labels.
+12. Click `Authorize APIs` and You will be in Step 2.
```

Comment:
```suggestion
12. Click `Authorize APIs`, and You will be in Step 2.
```

Reply:
Fixed in [a718fba](https://github.com/ballerina-guides/integration-samples/pull/32/commits/a718fba876d7ed04e212bd0cd384c113178380cb)